api/desecapi/models.py

```diff
@@ -334,6 +334,10 @@ def _partitioned_name(self):
         subname, _, parent_name = self.name.partition('.')
         return subname, parent_name or None
 
+    @property
+    def axfr(self):
+        return pdns.get_axfr(self)
```

Comment:
Should we allow filtering some records, e.g. export non-DNSSEC records only? (People usually don't want to import RRSIGs elsewhere as they are going to expire soon.)

Reply:
nslord only returns non-DNSSEC records this way

Reply:
aha! Cool. I did my experiments with nsmaster.